### DISCUSSION TITLE
ESP32DEVKIT V1 doesn't show as BT device

### DISCUSSION THREAD

**Discussion by 3charry** [darthcloud/BlueRetro#975] [Q&A]
Hello!  I had a older version and I did a upgrade from OTA, and everything was ok. I was playing in my PS2 and works properly but when I went to use again stopped to work and my DS3 couldn't connect. I tried to connect on BlueRetro.io, but doesn't show. I'm newbie so I tried to flash again the older version (And a newer one) finding a solution, but I still on the same trouble. 
My ESP32DEVKIT V1 is detected on my PC (Windows 10), but only shows a solid red light, I cannot see a blue. I tried flash the last HW1 and his 4 files, Flash Download Tool said "Finish" but I can't connect. I tried using the physical buttons, but nothing happens. What should I try next?

**Comment by darthcloud** [comment]
Try to see if you see some log on serial port:
https://github.com/darthcloud/BlueRetro/wiki/Getting-BlueRetro-debug-logs-via-Serial-port-Windows-10

If not then your ESP32 is improperly flash.
